corehq/apps/userreports/reports/view.py

```diff
@@ -121,7 +121,11 @@ def query_dict_to_dict(query_dict, domain, string_type_params):
     for key in string_type_params:
         u_key = str(key)  # QueryDict's key/values are unicode strings
         if u_key in query_dict:
-            request_dict[key] = query_dict[u_key]  # json_request converts keys to strings
+            vals = query_dict.getlist(u_key)
+            if len(vals) > 1:
+                request_dict[key] = vals
+            else:
+                request_dict[key] = vals[0]
```

Comment:
Trying to think if there's any place this assumption wouldn't be valid. I took a more surgical approach in https://github.com/dimagi/commcare-hq/pull/25371/commits/3175af40eb7b49900e1d2698b8f846d26d856248

But I think this is probably ok.

Reply:
Frankly I just took the work that had already been done on this ticket and tested it, I'm fairly unfamiliar with UCRs and wasn't eager to make more changes in this code. I agree your PR is a more deliberate change, if you want to deploy it on top of this (given that you're deploying the other cleanup work in that branch) please go for it.

Reply:
Reading [the docs for `getlist`](https://docs.djangoproject.com/en/2.2/ref/request-response/#django.http.QueryDict.getlist) I think this should be safe and I like that it's much simpler than my version so seems good. I was just being cautious since I suspect these filters don't have great test coverage.